circle.yml

```diff
@@ -7,13 +7,15 @@ dependencies:
   override:
     - pip install -r requirements.txt
     - pip install -r requirements_tests.txt
-test:
-  pre:
     - python setup.py install
```

Comment:
Running `python setup.py install` as a pre-test step only runs it in a single container. I moved it to `dependences/override` so it's run on all containers.